editor/modes/visual-editor/block.js

```diff
@@ -208,6 +208,7 @@ class VisualEditorBlock extends wp.element.Component {
 
 	onPointerDown() {
 		this.props.onSelectionStart();
+		this.props.onSelect();
```

Comment:
Do we need to select the block here? Isn't there some redundancy between this and the `onFocus` handler of the block root return element?

Reply:
As far as I can see, there's indeed no need fo this. Everything seems to work correctly. Are you running into issues?

Reply:
Yeah, there were some issues with nested events in #3745. Needed to remove this line in the latest changes there.

Reply:
Looking again at this, there may be one edge case where we'd want this, since there are a few situations in which the block becomes selected as currently implemented:

1. When the block calls its own `setFocus` (e.g. via `Editable`)
2. When the block wrapper is directly focused (e.g. clicking on the edge surrounding a block)
3. ...when otherwise clicking on a block (the logic implemented here)

So if we drop the third, it means a block could only become selected if either it sets its own focus by `setFocus` or the user clicks on the edges of the block, but as far as I can tell, it would not cause the block to become selected in other cases when clicking within the block.

I'm going to spend a bit more time debugging _why_ this is causing an issue for nesting event bubbling before deciding whether we should try to drop it.